### ISSUE TITLE
fetch-vendor-resources.sh fails to run (google-font-download)

### ISSUE THREAD

**Issue by peterkaminski** [GovReady/govready-q#1102]
When `fetch-vendor-resources.sh` uses this version of `google-font-download`:

```
https://raw.githubusercontent.com/neverpanic/google-font-download/ba0f7fd6de0933c8e5217fd62d3c1c08578b6ea7/google-font-download
```

it fails with these errors, from `google-font-download`:

```
Downloading Hind:400...Failed to determine local font name
Downloading Hind:700...Failed to determine local font name
Downloading Lato:900...Failed to determine local font name
```

(Observed by David A. on Windows/WSL and by Pete on macOS.)

Using a more recent version of `google-font-download`:

```
https://raw.githubusercontent.com/neverpanic/google-font-download/d7bb34e0c5fa01d9411213fd1a1363317d3ceb11/google-font-download
```

appears to work.

**Comment by davidpofo** [comment]
Tried out the changes to fetch-vendor-resources.sh and it downloaded and found the fonts as desired.

**Comment by gregelin** [comment]
I think this is getting fixed in PR #1066
